lib/cadet_web/controllers/grading_controller.ex

```diff
@@ -4,6 +4,20 @@ defmodule CadetWeb.GradingController do
 
   alias Cadet.Assessments
 
+  def index(conn, %{"group" => group}) do
+    user = conn.assigns[:current_user]
+
+    case Assessments.all_submissions_by_grader(user, group) do
+      {:ok, submissions} ->
+        render(conn, "index.json", submissions: submissions)
+
+      {:error, {status, message}} ->
+        conn
+        |> put_status(status)
+        |> text(message)
+    end
+  end
+
   def index(conn, _) do
```

Comment:
This code is not DRY. Instead what you can do is:
```elixir
def index(conn, _) do
  index(conn, %{"group" => false})
end
```

Reply:
Yup done!